Paws4Me/Paws4Me/Extensions/UITableViewSectionLabelExtension.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  UITableViewSectionLabelExtension.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/04/01.
+//
+
+import Foundation
+import UIKit
+
+extension UILabel {
+
+    func styleSectionLabel() {
+    self.font = UIFont(name: "Verdana", size: 20)
+    self.backgroundColor = UIColor.systemGray5
+    self.textColor = UIColor.black
```

Comment:
Think this should already be the default?

Reply:
That is true 👍🏻